config/config.php

```diff
@@ -2,9 +2,11 @@
 //The main config file
 define('BASE_URL', '');
 define('ENV', 'dev');
+define('MODELDEBUG', 'false');
```

Comment:
make this an actual boolean and not a string, since you replace an actual boolean in `Model.php`